client/src/pages/SearchResult/SingleSearchResult.tsx

```diff
@@ -60,6 +69,18 @@ export function SingleSearchResultPage() {
   // default should be no, not favorited. use boolean type. True is selected.
   const [heart, setHeart] = useState<boolean>(false);
   const prevState = "";
+  // dollar
+  let dollarAPI = restaurantData.price_level;
+  console.log("dollarAPI: " + dollarAPI);
+
+  const [price, setPrice] = useState(0);
```

Comment:
We don't need line 76 anymore since its being set on line 73 from the restaurantData object